src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -846,3 +956,11 @@ cdef class AioServer:
                 self._status
             )
         shutdown_grpc_aio()
+
+    cdef thread_pool(self):
+        """Access the thread pool instance."""
+        if self._thread_pool:
```

Comment:
Nit: I'm not sure I understand why this method can't just be `return self._thread_pool`. What falsey value could it have besides `None`?

Reply:
Simplified. Good catch! The else clause used to return another executor, until I found `asyncio. run_in_executor ` accepts `None`.